local/include/etc/confluent/docker/configureDefaults

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare -A env_defaults
+env_defaults=(
+  ["CLUSTER_ID"]="4L6g3nShT-eMCtK--X86sw"
```

Comment:
why are we hard coding these values?

Reply:
This was an ask from the product side to add a functionality where we are providing default configs as well so that developer can bring up the container directly using docker run -P <image>. Mentioned above(https://github.com/confluentinc/kafka-images/pull/211#issuecomment-1551520386)

Reply:
Can you put a comment similar to what exists in https://github.com/confluentinc/cp-all-in-one/blob/aeb622ca70498cd0dfb95bc114951714aa9af869/cp-all-in-one-kraft/docker-compose.yml#LL28C7-L30C43 so that it can be overridden?